docs/samples/Microsoft.ML.Samples/Dynamic/ImageClassification/ResnetV2101TransferLearningTrainTestSplit.cs

```diff
@@ -292,6 +328,8 @@ public class ImageData
 
             [LoadColumn(1)]
             public string Label;
+
+            public VBuffer<byte> ImageVBuf;
```

Comment:
>ImageVBuf [](start = 33, length = 9)

Please rename to ImageBuffer. VBuffer is the internal structure used to hold buffers that can be reused.

Reply:
Changed.